src/authy/core.clj

```diff
@@ -81,8 +80,8 @@
                        :socket-timeout   timeout
                        :throw-exceptions false})
            (:body)
-           (:error)
-           (= "invalid_grant")))
+           (:status)
+           (= "error")))
```

Comment:
Tapis now returns a response like the following:
```clojure
{:message "Invalid username/password combination.",
 :metadata {},
 :result nil,
 :status "error",
 :version "dev"}
```
I just updated this function to check for the error status, but I'm not sure if we need to also check the message.

Reply:
Hmm, yeah, that's a good question. It might be useful to log the message if the status happens to be equal to `error`, but I wouldn't expect us to need it for anything else.